Mathlib/Analysis/Calculus/ParametricIntegral.lean

```diff
@@ -247,6 +249,129 @@ theorem hasFDerivAt_integral_of_dominated_of_fderiv_le'' [NormedSpace ℝ H] {μ
 
 section
 
+open Real ContinuousLinearMap Asymptotics Interval
+open Set
+
+variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E] [CompleteSpace E]
+  {H : Type*} [NormedAddCommGroup H] [NormedSpace ℝ H]
+
+theorem hasFDerivAt_parametric_primitive_of_lip' (F : H → ℝ → E) (F' : ℝ → H →L[ℝ] E) {x₀ : H}
```

Comment:
I'm not sure what this is primed, as I didn't find a similarly named result in the library.

Also, the statement of this is sufficiently complicated that it *definitely* needs a good docstring.